### PR TITLE
separate actions

### PR DESCRIPTION
**Pull request by dpo**

### CODE REVIEW

File: .github/workflows/build-latex.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Build LaTeX document
 on:
   push:
     branches:
-      - main
+      - master
```
Comment (amontoison) [inline .github/workflows/build-latex.yml:6]:
```suggestion
      - main
      - master
```

Comment (amontoison) [reply .github/workflows/build-latex.yml:6]: It will work with old and new GitHub repositories :)

Comment (dpo) [reply .github/workflows/build-latex.yml:6]: Cool. I just copied the files the MInAres repo :-)

**Comment by dpo** [comment]
In fact the `latex-diff` action shouldn't be run on `push`.

File: .github/workflows/build-latex-diff.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+name: Build LaTeX diff document
+on:
+  push:
```
Comment (dpo) [inline .github/workflows/build-latex-diff.yml]:
```suggestion
```

File: .github/workflows/build-latex-diff.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    branches:
```
Comment (dpo) [inline .github/workflows/build-latex-diff.yml]:
```suggestion
```

File: .github/workflows/build-latex-diff.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      - master
```
Comment (dpo) [inline .github/workflows/build-latex-diff.yml]:
```suggestion
```

**Pull request merged**